### PR TITLE
add 12 new triggers from Ontologies issue #114

### PR DESCRIPTION
**Pull request by zupon**
Adds 12 new triggers gathered from BBN's clusters.

Resolves WorldModelers/Ontologies#114.

### CODE REVIEW

**Comment by MihaiSurdeanu** [comment]
@zupon: these make sense to me, but tests fail. Can you please look at this?
@kwalcock 's manual for how to navigate jenkins might help. @kwalcock: where is that again? I don't see it under the clulab Wiki.

Thanks!

**Comment by kwalcock** [comment]
It's where I would expect it at https://github.com/clulab/clulab/wiki/Jenkins, now with working internal links.

**Comment by MihaiSurdeanu** [comment]
Thanks!

On March 7, 2021 at 6:48:14 PM, Keith Alcock (notifications@github.com)
wrote:

It's where I would expect it at
https://github.com/clulab/clulab/wiki/Jenkins, now with working internal
links.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/clulab/eidos/pull/980#issuecomment-792406138>, or
unsubscribe

File: src/main/resources/org/clulab/wm/eidos/english/grammars/triggers.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ increase_triggers:
   - boost
   - compel(s?$)|compell
   - compound
+  - cultivat
```
Comment (BeckySharp) [inline src/main/resources/org/clulab/wm/eidos/english/grammars/triggers.yml:10]: I think we need to make this be a verb only, so like `cultivat[e|ing]`

**Review by BeckySharp — COMMENTED** [review]
I think this may take care of things -- I'll check

**Comment by BeckySharp** [comment]
This is now passing, should we merge it @MihaiSurdeanu and @kwalcock ?

**Review by BeckySharp — APPROVED** [review]
I approve my changes :D

**Comment by kwalcock** [comment]
Yes, I approve.  It's fine to observe that the silent e is losing favor.

**Pull request merged**